extensions/ql-vscode/src/helpers.ts

```diff
@@ -232,6 +232,15 @@ export function getOnDiskWorkspaceFolders() {
   return diskWorkspaceFolders;
 }
 
+/** Check if folder is already present in workspace */
+export function isFolderAlreadyInWorkspace(folderName: string) {
+  const workspaceFolders = workspace.workspaceFolders || [];
```

Comment:
Maybe we only need to look in the list of folders from `getOnDiskWorkspaceFolders()`? I don't fully understand what that function does, but we use it in a lot of places 😅